next-frontend/src/app/(wca)/competitions/[competitionId]/live/competitors/[registrationId]/page.tsx

```diff
@@ -0,0 +1,88 @@
+import { Container, Heading, Link, Table } from "@chakra-ui/react";
+import { getResultByPerson } from "@/lib/wca/live/getResultByPerson";
+import _ from "lodash";
+import events from "@/lib/wca/data/events";
+import { rankingCellStyle } from "@/components/live/LiveResultsTable";
+import { centisecondsToClockFormat } from "@/lib/wca/wcif/attempts";
+
+export default async function PersonResults({
+  params,
+}: {
+  params: Promise<{ registrationId: string; competitionId: string }>;
+}) {
+  const { competitionId, registrationId } = await params;
+
+  const personResultRequest = await getResultByPerson(
+    competitionId,
+    registrationId,
+  );
+
+  if (!personResultRequest.data) {
+    return <p>Something went wrong while trying to fetch results</p>;
+  }
+
+  const { name, results } = personResultRequest.data;
+
+  const resultsByEvent = _.groupBy(results, "event_id");
+
+  return (
+    <Container>
+      <Heading size="5xl">{name}</Heading>
+      {_.map(resultsByEvent, (eventResults, key) => (
+        <>
+          <Heading size="2xl">{events.byId[key].name}</Heading>
```

Comment:
Rather than heading sizes, you should work with `textStyle`s defined in our Chakra theme. We have `h1` until `h3`, and `s1` and `s2`. Play around and see which one suits you best.